### PR TITLE
Tour for 7.2 

### PR DESCRIPTION
**Pull request by Rajat-Dabade**
#### Summary
This PR will add a tour point for 7.2 release

Card link - https://community.mattermost.com/boards/team/qghzt68dq7bopgqamcnziq69ao/b4ctmm917q3rzdfi451qtuxjzcw/vd89yph84ajbkufkbgts9m3sb5a/ckz6bqk4zotdmdrwzo14r7su9kr

### CODE REVIEW

File: webapp/src/components/sidebar/sidebarCategory.tsx [hunk 1]
```diff
@@ -248,10 +299,25 @@ const SidebarCategory = (props: Props) => {
...
                 )
             })}
-            {!collapsed && props.boards.map((board: Board) => {
+            {!collapsed && props.boards.map((board: Board, index) => {
                 if (!isBoardVisible(board.id)) {
                     return null
                 }
+                if(index === 0) {
+                    return (
+                        <SidebarBoardItem
```
Comment (harshilsharma63) [inline webapp/src/components/sidebar/sidebarCategory.tsx:308]: The only difference betwene the if and else block is with `shouldViewManageBoardsTour` prop being passed in one of those. We can avoid the if else and just have one SidebarBoardItem with `shouldViewManageBoardsTour={index === 0 ? shouldViewManageBoardsTour : undefined}`

**Review by harshilsharma63 — CHANGES_REQUESTED** [review]

**Comment by harshilsharma63** [comment]
The spacing between end of text and next button is very low. Can you use the same spacing we have in other tour components?
Screenshot 2022-08-10 at 4 16 58 PM

File: webapp/src/components/onboardingTour/sidebarCategories/sidebarCategories.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    useEffect(() => {
+        async function task() {
+            if (!me) {
+                return
+            }
+
+            const should = onboardingTourCategory === TOUR_BOARD &&
+                           onboardingTourStep === FINISHED.toString()
+
+            console.log("Shoould: ", should, "isOnboardingBoard", isOnboardingBoard, "onBoardingTourCategory", onboardingTourCategory, "onboardingTourStep", SidebarTourSteps.SIDE_BAR)
```
[the remark this answers is not in the document]
Comment (Rajat-Dabade) [reply webapp/src/components/onboardingTour/sidebarCategories/sidebarCategories.tsx:62]: Sure, will remove it.

File: webapp/src/components/onboardingTour/tourTipRenderer/tourTipRenderer.tsx [hunk 3]
```diff
@@ -28,17 +28,24 @@ type Props = {
...
-    const isOnboardingBoard = board ? board.title === OnboardingBoardTitle : false
+    let isOnboardingBoard = board ? board.title === OnboardingBoardTitle : false
     const onboardingTourStarted = useAppSelector(getOnboardingTourStarted)
     const onboardingTourCategory = useAppSelector(getOnboardingTourCategory)
     const onboardingTourStep = useAppSelector(getOnboardingTourStep)
     const disableTour = clientConfig?.featureFlags?.disableTour || false
+
+
+    if(props.showForce) {
+        isOnboardingBoard = props.showForce ? true: false
```
[the remark this answers is not in the document]
Comment (Rajat-Dabade) [reply webapp/src/components/onboardingTour/tourTipRenderer/tourTipRenderer.tsx:46]: Perfect will make the changes.

File: webapp/src/components/sidebar/sidebarCategory.tsx (hunk 1 above)
Comment (Rajat-Dabade) [reply webapp/src/components/sidebar/sidebarCategory.tsx:308]: Correct will update this.

**Comment by Rajat-Dabade** [comment]
@harshilsharma63, Thanks for the feedback. the space was because of the gif margin we were using in other tour steps, I have explicitly added the margin for the footer. Which will solve the spacing problem.

**Review by sbishel — APPROVED** [review]

**Comment by sbishel** [comment]
@Rajat-Dabade - Please make sure tags and milestone are added, when appropriate.

**Pull request merged**